gcampus/core/models/measurement.py

```diff
@@ -254,3 +253,28 @@ def __str__(self):
             "pk": self.pk,
             "name": self.parameter_type.name,
         }
+
+
+class Calibration(models.Model):
+    class Meta:
+        verbose_name = _("Calibration")
+        verbose_name_plural = _("Calibrations")
+
+    name = models.CharField(blank=True, max_length=280, verbose_name=_("Name"))
+
+    calibration_parameter_type = models.ForeignKey(
+        ParameterType,
+        on_delete=models.PROTECT,
+        # We opted to use 'Parameter' in the front-end. See issue
+        # https://github.com/desklab/gcampus/issues/46 for more.
+        verbose_name=_("Parameter"),
+        related_name="calibration_parameter",
+    )
+
+    calibration_formula = models.CharField(
+        blank=True, max_length=100, verbose_name=_("Calibration formula")
+    )
+
+    x_max = models.FloatField(default=-9999, verbose_name=_("Maximal x value"))
```

Comment:
What is `x` in this context? I think the field name (`x_max` and `x_min`) is fine, but if x is the OD, the verbose name should maybe say _"Maximal OD value"_.

If x corresponds to the parameter type (say x is for example the pH value), you can leave it this way 😉

Reply:
fixed in 067baa0b47adfd6587eff8d377fe351906522832